### DISCUSSION TITLE
Anyone know why the Fault-Tolerant Training document was removed from the latest pytorch-lighting document?

### DISCUSSION THREAD

**Discussion by Keiku** [Lightning-AI/pytorch-lightning#17811] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
Anyone know why the Fault-Tolerant Training document was removed from the latest pytorch-lighting document? It seems to have existed at least as of version 1.7.7.

**Comment by justusschock** [comment, marked as answer]
Hey, this feature was removes with PL 2.0.0 as it was having to high impact on complexity and thereby training speed. You can still manually do this by saving the random seed of your dataloaders/datasets and restoring them upon loading.

**Reply by Keiku** [reply]
@justusschock Thank you for your answer. I am looking for a refactor part in the **_Lightning League Refactored Project_**, but could we consider adding the manual Fault-Tolerant Training mentioned in the latter part to the document or example? I think the Lighting AI team may remove that part when it's really done.
